### ISSUE TITLE
Linking users to country or region

### ISSUE THREAD

**Issue by kitenetter** [NERC-CEH/fit-count-app#144]
I think we've discussed this before, but it has arisen again today. I need to produce a mailing list for all UK FIT Count contributors who have given permission to be contacted. However, there is no mechanism within the website user accounts for users to specify which country they are participating in.

So the only way I can produce a UK mailing list is to link website accounts to samples in the warehouse that are from within the UK. This is time-consuming and also misses out on any UK people who have registered, and may wish to receive communications, but have not yet contributed a count. 

Presumably the same problem will arise for any other countries who wish to contact the users who have given permission. to be contacted.

Is there a solution to this issue? Do we need to extend the registration data so that users specify which country they are active in?

**Comment by DavidRoy** [comment]
@kazlauskis - can you capture the country selected by the app users when they setup the app?

Or the alternative as Martin suggests is asking this question on registration

**Comment by kazlauskis** [comment]
I agree. We should use the country choice from the app onboarding. This would could then be uploaded to the website when the user registers or logins.

1. App onboarding - user chooses a country
2. Registration - the country is set to the user's website profile. We don't ask to specify the country again in the form, it is set in the background.

For any existing users, we could also upload this value in the background to the website profile. This would then also allow changing the country in the app which would then automatically update on the website. 

This **app→website** sync should be fairly easy to do if we can avoid the opposite **website→app** sync. The app is the primary way to interact with the system so I doubt many users would be changing countries on the website - I would probably hide this field editing there.

**Comment by kitenetter** [comment]
@kazlauskis I agree with the approach you suggest, and I don't see any need to sync from the website to the app.

**Comment by DavidRoy** [comment]
Thanks @kazlauskis.  Please can you add this to your schedule

**Comment by JimChiazzese1** [comment]
Hi @kazlauskis - Andy alerted me to this item, which I wasn't aware of. This is in discussion currently as Ireland want to mail out to 'Ireland' FITCount users to alert them to the fact that it's pollination monitoring season coming up. 
Is this is a task that's feasible in the coming weeks and if so, do you have an idea of budget and effort required? 

My understanding is that we'd then have a system for sorting users by their selected 'country' rather than just because they submitted a count in that country when they may have been visiting only. Thanks Jim

**Comment by kazlauskis** [comment]
Hi @JimChiazzese1, we won't be able to work on this just yet but towards the end of next month we should be ready to pick up new work. We would need around 3 days to do it, that's £1k. If you'd like us to work on this then assign the ticket to the next app's milestone so we don't forget it. Cheers

**Comment by kitenetter** [comment]
Also discussed under #186
